core/rio/turtle/src/main/java/org/eclipse/rdf4j/rio/turtle/TurtleUtil.java

```diff
@@ -459,6 +459,17 @@ public static String encodeURIString(String s) {
 		return s;
 	}
 
+	public static boolean isValidPrefixedName(String s) {
+		final int numberOfCodePoints = s.codePointCount(0, s.length());
+		for (int i = 1; i < numberOfCodePoints; i++) {
+			final int codePoint = s.codePointAt(i);
+			if (!isPN_CHARS(codePoint)) {
+				return false;
+			}
+		}
+		return true;
+	}
+
```

Comment:
Maybe you could use the codepoints() method?

```
public static boolean isValidPrefixedName(String s) {
    return s.codePoints().skip(1) // Skip the first code point
           .allMatch(TurtleUtil::isPN_CHARS);
}
```

Reply:
Will be included in PR update.